node/completion.bash

```diff
@@ -0,0 +1,4 @@
+export NVM_DIR=/usr/local/var/nvm
+
+nvm_completion="$(brew --prefix nvm)/nvm.sh"
```

Comment:
Please use following structure:
* nvm/config.bash - to load nmv
* nvm/completion.bash - to setup bash completion